src/power_grid_model/data_types.py

```diff
@@ -11,12 +11,10 @@
 
 import numpy as np
 
-from power_grid_model.core.dataset_definitions import ComponentType
-
-ComponentTypeVar = TypeVar("ComponentTypeVar", ComponentType, str)
+from power_grid_model.core.dataset_definitions import ComponentTypeLike  # pylint: disable=unused-import
```

Comment:
What's the purpose of importing btw?

Reply:
interface

Reply:
no, correction, you are right: we want them to use `ComponentType` or if really needed they can specify their own type, in which case it's entirely on them. will remove